mailbox/plugin/spamassassin/pom.xml

```diff
@@ -77,6 +77,10 @@
             <artifactId>testing-base</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.google.inject</groupId>
+            <artifactId>guice</artifactId>
```

Comment:
I guess you're thinking it's suck. Suggestions are welcome

Reply:
don't use @ImplementedBy